framework/source/class/qx/event/handler/Focus.js

```diff
@@ -1075,9 +1075,13 @@ qx.Class.define("qx.event.handler.Focus",
      */
     __fixFocus : qx.event.GlobalError.observeMethod(qx.core.Environment.select("engine.name",
       {
-        "mshtml" : this.__getCorrectFocusTarget,
+        "mshtml" : function(target) {
+          return this.__getCorrectFocusTarget(target);
+        },
 
-        "webkit" : this.__getCorrectFocusTarget,
```

Comment:
@derrell as this happens on class construction I'm not sure if `this` is valid here.